docs/pages/example/create-color-ramp-for-building-layer.js

```diff
@@ -0,0 +1,10 @@
+/*---
+title: Create color and opacity ramp for building layer
+description: Uses exponential interpolation to ease-in the building layer and smoothly fade from one color to the next.
```

Comment:
Let's explicitly call out that this is an example using expressions and link to those docs here.